recipes/jupyterhub/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.1" %}
+
+package:
+  name: jupyterhub
+  version: {{ version }}
+
+source:
+  fn: jupyterhub-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/ac/9b/ef3435cf8951d7463f4175a0066a8454e224a598b2159f521cc011f8c2b6/jupyterhub-{{ version }}.tar.gz
+  md5: 6bd795a82bf15a037d097991110b8713
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: pip install --no-deps .
```

Comment:
> but it would be good if we can get everyone to agree

Sure we need agreement on important decisions. However, I don't think this is major and we already merged feedstocks with that syntax. So I think this ship has sailed.

> The issue we have now is if everyone is building each package differently it is more likely some issues creep in or we miss something during a review due to all of the acceptable ways to do something.

I don't think that is an issue. I am OK with 1-2 different forms for `script`. And, if anyone is found broken, we have to fix it. Right now we know that `pip` is not.

> No one wants broken packages.

Agreed!